<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableSelect</class>
 <widget class="QWidget" name="TableSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>138</width>
    <height>46</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="hboxMain">
   <property name="spacing">
    <number>12</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="fraDirTable">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout" name="gridDirTable">
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lblDirImage">
        <property name="toolTip">
         <string>Directory Table</string>
        </property>
        <property name="pixmap">
         <pixmap resource="resources/oxygen/oxygen.qrc">:/oxygen/16x16/document-open.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnDirA">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string notr="true">A</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btnDirB">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string notr="true">B</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="lblDirAStatus">
        <property name="pixmap">
         <pixmap resource="resources/oxygen/oxygen.qrc">:/oxygen/16x16/dialog-ok-apply.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="lblDirBStatus">
        <property name="pixmap">
         <pixmap resource="resources/oxygen/oxygen.qrc">:/oxygen/16x16/dialog-close.png</pixmap>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="fraBlockTable">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout" name="gridBlockTable">
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item row="1" column="1">
       <widget class="QPushButton" name="btnBlockB">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string notr="true">B</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblBlockImage">
        <property name="toolTip">
         <string>Block Table</string>
        </property>
        <property name="pixmap">
         <pixmap resource="resources/oxygen/oxygen.qrc">:/oxygen/16x16/document-save-all.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnBlockA">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>23</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string notr="true">A</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="lblBlockAStatus">
        <property name="pixmap">
         <pixmap resource="resources/oxygen/oxygen.qrc">:/oxygen/16x16/dialog-ok-apply.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="lblBlockBStatus">
        <property name="pixmap">
         <pixmap resource="resources/oxygen/oxygen.qrc">:/oxygen/16x16/dialog-close.png</pixmap>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources/oxygen/oxygen.qrc"/>
 </resources>
 <connections/>
</ui>
